test/util/exec.spec.ts

```diff
@@ -3,107 +3,214 @@ import {
   ExecOptions as ChildProcessExecOptions,
 } from 'child_process';
 import { exec, ExecOptions } from '../../lib/util/exec';
+import { envMock } from '../execUtil';
 
 const cpExec: jest.Mock<typeof _cpExec> = _cpExec as any;
-jest.mock('child_process');
 
-const image = 'example/image';
+jest.mock('child_process');
 
-const tag = '1.2.3';
+interface TestInput {
+  processEnv: Record<string, string>;
+  inCmd: string;
+  inOpts: ExecOptions;
+  outCmd: string;
+  outOpts: ChildProcessExecOptions & { encoding: string };
+  trustLevel?: 'high' | 'low' | unknown;
+}
 
-const cmd = 'echo hello';
+describe(`Child process execution wrapper`, () => {
+  let processEnvOrig;
+  let trustLevelOrig;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    jest.resetModules();
+    processEnvOrig = process.env;
```

Comment:
Can we use the env mock here too?

Reply:
No, here we need to test the lower level

Reply:
ok